### ISSUE TITLE
Add Page SEO 

### ISSUE THREAD

**Issue by princemuel** [princemuel/invoice-web-app#64]
Necessary steps should be taken to improve aspects of the application such as page load speed, SEO friendly html tags with semantics and accessibility in mind, overall user interactivity and experience etc.
